index.html

```diff
@@ -20,11 +20,105 @@
           <h1 class="box box_1">Hey there. I'm <br>Sumair</h1>
           <h2 class="box box_2">I'm a software<br>developer</h2>
           <p class="box box_3">I can help you build a product,<br> feature or website. Look through<br> some of my work and experience!<br> If you like what you see and have<br> a project you need coded,<br> don’t hesitate to contact me.</p>
-            <ul class="web_buttons">
+            <ul class="web_button">
               <li class="web_buttons_1"><a  href="https://twitter.com/sumair_qaisar"><img src="assets/images/social_media/twitter.svg" alt="Twitter icon"></a></li>
               <li class="web_buttons_2"><a href="https://linkedin.com/in/sumair-qaisar-jadoon-84a877164"><img src="assets/images/social_media/linkedin.svg" alt="Linkedin icon"></a></li>
               <li class="web_buttons_3"><a href="https://github.com/sumairq"><img src="assets/images/social_media/github.svg" alt="Github icon"></a></li>
             </ul>
+            <a id="down_icon" href="#"><img src="assets/images/down.png" alt="Icon for moving"></a>
       </section>
+      <header>
+        <h2 class="text box_1">My Recent Works </h2>
+        <img class="center" src="assets/images/Indicator.png" alt="Design line">
+
+      </header>
+
+      <!-- main grid container -->
+      <section class="grid_container">   
+        <!-- flexbox container   -->
+        <div class="flex_container_primary">
```

Comment:
- [ ] [OPTIONAL]: To keep your HTML semantic, I suggest you replace the `<div>` containers with `<article>` tags. You can read more about article tags [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).